pkg/portForward/portForward.go

```diff
@@ -106,6 +103,23 @@ func (o *PFClient) StopPortForwarding() {
 	o.finishedChan = nil
 }
 
+func (o *PFClient) GetForwardedPorts() map[string][]int {
+	return o.appliedEndpoints
```

Comment:
Do we need a method for just one statement? And it's already a part of the struct.

Reply:
I like to hide this kind of contents of structs so the caller is not tempted to edit these fields himself